exporter/signalfxexporter/config.go

```diff
@@ -104,6 +104,9 @@ type Config struct {
 	// NonAlphanumericDimensionChars is a list of allowable characters, in addition to alphanumeric ones,
 	// to be used in a dimension key.
 	NonAlphanumericDimensionChars string `mapstructure:"nonalphanumeric_dimension_chars"`
+
+	// MaxConnections is used to set a limit to the maximum idle HTTP connection the exporter can keep open.
+	MaxConnections int `mapstructure:"max_connections"`
```

Comment:
It would be nice to refactor signalfx Config struct to use [HTTPClientSettings](https://github.com/open-telemetry/opentelemetry-collector/blob/45a848d8abe14cd58dd937de7a257a6721d19281/config/confighttp/confighttp.go#L36) and this option there so that all other http-based exporters benefit uniformly.

Can be done in a future PR.

Reply:
Created a new PR